### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TaeBlack**
update v1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,60 @@
...
+# ![Pyrrha](/img/prometeo.png?raw=true)
 
-In the face of natural disasters, such as wildfires, Prometeo protects those who protect us - the firefighters. With our solution, we've developed a prototype sensor which sends environmental telemetry (temperature, humidity, and smoke concentration). This data is processed by a machine learning algorithm which is able to predict firefighter health, indicating if they are OK, soon to be in danger, or in danger. The goal is to track real-time status, and eventually use aggregated data to make suggestions that improve health outcomes in the long run.
+In the face of natural disasters, such as wildfires, Pyrrha (created by Prometeo) protects those who protect us - the firefighters. With our solution, we've developed a prototype sensor which sends environmental telemetry (temperature, humidity, and smoke concentration). This data is processed by a machine learning algorithm which is able to predict firefighter health, indicating if they are OK, soon to be in danger, or in danger. The goal is to track real-time status, and eventually use aggregated data to make suggestions that improve health outcomes in the long run.
+
+
+## Pyrrha Team
```
Comment (upkarlidder) [inline README.md:8]: Add core Prometeo team
- Marco
- Salome
- Joan
- TBD
@upkarlidder to provide names

File: README.md [hunk 2]
```diff
@@ -1,20 +1,60 @@
...
+| Name | Title | Email | Slack |
+| --- | --- | --- | --- |
+| Upkar Lidder | Technical Lead | Ulidder@us.ibm.com | @Upkarlidder |
```
Comment (upkarlidder) [inline README.md:11]: Remove all emails. Add GitHub profile links.

[Upkar Lidder](https://github.com/upkarlidder)

File: README.md [hunk 3]
```diff
@@ -1,20 +1,60 @@
...
+| Daniel Krook | CTO | Krook@us.ibm.com | @Krook |
+| John Segrave | Lead Data Scientist & Technical Product Manager | John.segrave@ie.ibm.com | @Jsegrave |
+| Marco Emilio Rodriguez Serrano | --- | Marco.Rodriguez.Serrano@ibm.com | @Marco.rodriguez.serra |
+| Salome Valero Cumplido | --- | Salome.valero@es.ibm.com | @Salaome.valero |
+| Taeson Black | Tech. Solution Specialist Intern | Tae.Black@ibm.com | @Tae.black|
+| Alejandro Quijada | Tech. Solution Specialist Intern | Alejandro.Quijada@ibm.com | @Alejandro.Quijada |
+
+## IBM Call for Code Team
```
Comment (upkarlidder) [inline README.md:19]: Check with @krook if we need separate section for CfC team.

File: README.md [hunk 4]
```diff
@@ -1,20 +1,60 @@
...
+| Name | Title | Email | Slack |
+| --- | --- | --- | --- |
+| Daniel Krook | CTO | krook@us.ibm.com | @Daniel Krook |
+| Gaurav Ramakrishna | Software Developer | gaurav.ramakrishna@ibm.com | @Gaurav Ramakrishna |
+| Edrian Irizarry | Senior Software Engineer | edrian@us.ibm.com | @Edrian Irizarry |
+
+## Other Partners
```
Comment (upkarlidder) [inline README.md:26]: Check with @krook

File: README.md [hunk 5]
```diff
@@ -1,20 +1,60 @@
...
+| Name | Title | Email | Slack |
+| --- | --- | --- | --- |
+| ? | ? | ? | ? |
+
+## Public Tools
+* Will use [Webex](https://ibm.webex.com/meet/gaurav.ramakrishna) for meetings - Weekly, starting with January 27 at 10pm Mongolia Time / 9am US Eastern Time
```
Comment (upkarlidder) [inline README.md:32]: Only keep
- slack workspace and channel # prometeo-pyrrha

File: README.md [hunk 6]
```diff
@@ -6,18 +6,17 @@ In the face of natural disasters, such as wildfires, Pyrrha (created by Prometeo
 
 
 ## Pyrrha Team
-| Name | Title | Email | Slack |
+| Name | Title | Github | Slack |
```
Comment (krook) [inline README.md:9]: Please capitalize to "GitHub"

File: README.md [hunk 7]
```diff
@@ -6,18 +6,17 @@ In the face of natural disasters, such as wildfires, Pyrrha (created by Prometeo
...
+| Upkar Lidder | Technical Lead | [Upkar](https://github.com/upkarlidder) | @Upkarlidder |
+| John Segrave | Lead Data Scientist & Technical Product Manager | [John](https://github.com/JSegrave-IBM) | @Jsegrave |
+| Marco Emilio Rodriguez Serrano | --- | [Marco](https://github.com/mrodrise) | @Marco.rodriguez.serra |
+| Salome Valero Cumplido | --- | [Salome](https://github.com/svaleroc) | @Salaome.valero |
+| Taeson Black | Tech. Solution Specialist Intern | [Tae](https://github.com/TaeBlack) | @Tae.black|
+| Alejandro Quijada | Tech. Solution Specialist Intern | [Alejandro](https://github.com/Aquijada52) | @Alejandro.Quijada |
 
 ## IBM Call for Code Team
-| Name | Title | Email | Slack |
+| Name | Title | github | Slack |
```
Comment (krook) [inline README.md:19]: Please capitalize to "GitHub"

**Review by krook — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (TaeBlack) [reply README.md:19]: so you do want seperate team blocks ; one for pyrrha, cfc , and other @krook

Comment (krook) [reply README.md]: Yes please

**Pull request closed without merging**